jdbc/src/main/java/org/apache/metamodel/jdbc/dialects/OracleQueryRewriter.java

```diff
@@ -18,17 +18,19 @@
  */
 package org.apache.metamodel.jdbc.dialects;
 
+import static org.apache.metamodel.jdbc.JdbcDataContext.DATABASE_PRODUCT_ORACLE;
+
 import org.apache.metamodel.jdbc.JdbcDataContext;
 import org.apache.metamodel.query.FilterItem;
 import org.apache.metamodel.schema.ColumnType;
 
 /**
  * Query rewriter for Oracle
  */
-public class OracleQueryRewriter extends DefaultQueryRewriter {
+public class OracleQueryRewriter extends OffsetFetchQueryRewriter {
 
     public OracleQueryRewriter(JdbcDataContext dataContext) {
-        super(dataContext);
+        super(dataContext, 11);
```

Comment:
Probably want to use a constant here to document the magic number, like `FIRST_FETCH_SUPPORTING_VERSION` (or maybe something a little less clumsy than what I came up with :smile:).